lib/mixins/template-stamp.html

```diff
@@ -313,6 +315,32 @@
        *   metadata to `nodeInfo`
        */
       static _parseTemplateNestedTemplate(node, outerTemplateInfo, nodeInfo) {
+        // If a `<dom-repeat>` or a `<dom-if>` is inside of an svg, we need
+        // to create all elements in a different namespace. To do so, make an svg,
+        // let the HTML parser parse in that namespace, and then put back all
+        // created children in a fresh template.
+        if (node.parentNode.localName && templateExtensions[node.parentNode.localName]
+              && node.parentNode.parentNode && node.parentNode.parentNode.namespaceURI === SVG_NAMESPACE) {
+          const svg = document.createElementNS(SVG_NAMESPACE, 'svg');
```

Comment:
I was picturing this fixup being done in `_parseTemplate` so it applies to any template.  But I see now that the `dom-repeat` case requires checking up two levels to find the SVG namespace (rather than just one), which requires the whitelist of template wrappers. I also just realized `dom-repeat` can't be used in the main document to repeat SVG content because the `dom-repeat` must be parsed using the `<template is="...">` syntax otherwise it'll be kicked out of the SVG during parsing.  Conclusion: I think this is fine for now to use the whitelist and only do in nested templates, as you've done.